featurebyte/query_graph/transform/offline_ingest_extractor.py

```diff
@@ -272,20 +281,30 @@ def _insert_offline_store_query_graph_node(
         request_columns = global_state.node_name_to_request_columns[node_name]
         parameter_class: Any
         if request_columns:
+            graph_node_type = GraphNodeType.OFFLINE_STORE_REQUEST_COLUMN_QUERY
             parameter_class = OfflineStoreRequestColumnQueryGraphNodeParameters
+            suffix = "__req_comp"
         else:
+            graph_node_type = GraphNodeType.OFFLINE_STORE_INGEST_QUERY
             parameter_class = OfflineStoreIngestQueryGraphNodeParameters
+            suffix = "__comp"
```

Comment:
nit: Slightly prefer the suffix to be a complete word such as `component` or `part`.